ue5osc/__init__.py

```diff
@@ -1,21 +1,108 @@
+from pathlib import Path
+from time import sleep
+
+from PIL import Image
 from pythonosc import udp_client
+from pythonosc.osc_server import BlockingOSCUDPServer
+import threading
+
+from ue5osc.osc_dispatcher import OSCMessageReceiver
+
+
+class Communicator:

```

Comment:
Did you have a chance to look into Python context managers?

Reply:
I have been reading some documentation on it, but haven't implemented it yet.

Reply:
It doesn't need to make it in this round. We can wait until later.